packages/components/src/components/tables/es-table/es-table.tsx

```diff
@@ -218,9 +225,9 @@ export class Table {
                     sticky={this.stickyHeader}
                 />
                 {this.loading &&
-                    new Array(this.loadingRows)
-                        .fill({})
-                        .map(this.renderRowGroup)}
+                    Array.from({ length: this.loadingRows }, (_, i) =>
+                        this.renderRowGroup({}, i),
+                    )}
                 {!this.loading && this.rows?.map(this.renderRowGroup)}
```

Comment:
Still think this would be clearer as a ternary, rather than checking `this.loading` twice.